sources/packages/backend/apps/api/src/route-controllers/note/_tests_/e2e/note.aest.controller.getInstitutionNotes.e2e-spec.ts

```diff
@@ -0,0 +1,190 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import { createFakeInstitution } from "@sims/test-utils";
+import { DataSource, Repository } from "typeorm";
+import { Institution, NoteType } from "@sims/sims-db";
+import {
+  AESTGroups,
+  createTestingAppModule,
+  BEARER_AUTH_TYPE,
+  getAESTToken,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+import {
+  createFakeNote,
+  saveFakeInstitutionNotes,
+} from "@sims/test-utils/factories/note";
+import { NoteAPIOutDTO } from "../../models/note.dto";
+
+describe("NoteAESTController(e2e)-getInstitutionNotes", () => {
+  let app: INestApplication;
+  let institutionRepo: Repository<Institution>;
+  let appDataSource: DataSource;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    appDataSource = dataSource;
+    institutionRepo = dataSource.getRepository(Institution);
+  });
+
+  it("Should allow access to the expected AEST users groups", async () => {
+    // Arrange
+    const institution = await institutionRepo.save(createFakeInstitution());
+    await saveFakeInstitutionNotes(
+      appDataSource,
+      [createFakeNote(NoteType.General)],
+      institution.id,
+    );
+    const endpoint = `/aest/note/institution/${institution.id}`;
+    const expectedPermissions = [
+      {
+        aestGroup: AESTGroups.BusinessAdministrators,
+        expectedHttpStatus: HttpStatus.OK,
+      },
+      {
+        aestGroup: AESTGroups.Operations,
+        expectedHttpStatus: HttpStatus.OK,
+      },
+      {
+        aestGroup: AESTGroups.OperationsAdministrators,
+        expectedHttpStatus: HttpStatus.OK,
+      },
+      {
+        aestGroup: AESTGroups.MOFOperations,
+        expectedHttpStatus: HttpStatus.OK,
+      },
+      {
+        aestGroup: undefined, // Read only user.
+        expectedHttpStatus: HttpStatus.OK,
+      },
+    ];
+
+    // Act/Assert
+    for (const permission of expectedPermissions) {
+      await request(app.getHttpServer())
+        .get(endpoint)
+        .auth(await getAESTToken(permission.aestGroup), BEARER_AUTH_TYPE)
+        .expect(permission.expectedHttpStatus);
+    }
+  });
+
+  it("Should get institution notes", async () => {
+    // Arrange
+    const institution = await institutionRepo.save(createFakeInstitution());
+    const [institutionNote] = await saveFakeInstitutionNotes(
+      appDataSource,
+      [createFakeNote(NoteType.General)],
+      institution.id,
+    );
+    const endpoint = `/aest/note/institution/${institution.id}`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
```

Comment:
The expected can be chaned as below.
```ts
await request(app.getHttpServer())
      .get(endpoint)
      .auth(await getAESTToken(), BEARER_AUTH_TYPE)
      .expect(HttpStatus.OK)
      .expect([
        {
          noteType: institutionNote.noteType,
          description: institutionNote.description,
          firstName: institutionNote.creator.firstName,
          lastName: institutionNote.creator.lastName,
          createdAt: institutionNote.createdAt.toISOString(),
        },
      ]);
```
Same idea from https://github.com/bcgov/SIMS/pull/1767/files#r1122188785